Formula/feedgnuplot.rb

```diff
@@ -16,7 +22,7 @@ class Feedgnuplot < Formula
   depends_on "gnuplot"
 
   def install
-    system "perl", "Makefile.PL", "prefix=#{prefix}"
+    system "perl", "Makefile.PL", "INSTALL_BASE=#{prefix}", "INSTALLSITEMAN1DIR=#{man1}", "INSTALLSITEMAN3DIR=#{man3}"
```

Comment:
unless I am wrong, there is no `man3` to install, right?

```suggestion
    system "perl", "Makefile.PL", "INSTALL_BASE=#{prefix}", "INSTALLSITEMAN1DIR=#{man1}"
```